test/common/testlib.py

```diff
@@ -924,28 +922,13 @@ def check_journal_messages(self, machine=None):
         if self.image in ['fedora-29', 'fedora-30', 'fedora-testing', 'fedora-i386', 'fedora-atomic']:
             # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1563143
             self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { getattr } for .* comm="which" path="/usr/sbin/setfiles".*')
-            # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1629588
-            self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { read } for .* comm="agetty" name="motd".*')
-            # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1662441
-            self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { getattr } for .* comm="find" path="/proc.*')
-            self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { mount } for .* comm="find" .*pcp_pmlogger_t.*')
             # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1662866
             self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { execute } for .* comm="which" .*pcp_pmlogger_t.*')
-            # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1406979
-            # The bug is closed and for old release but should be here as reference for the solutions
-            # It suggests configure auditd to dontaudit these messages since selinux can't offer whitelisting this directory for qemu process
-            self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { search } for  .* comm="qemu-.* dev="proc" .*')
-            self.allowed_messages.append('audit: type=1400 audit(.*): avc:  denied  { read } for  .* comm="qemu-.* dev="proc" .*')
 
         if self.image in ['fedora-30']:
             # Fedora 30 switched to dbus-broker
             self.allowed_messages.append("dbus-daemon didn't send us a dbus address; not installed?.*")
 
-        if self.image in ['rhel-8-0', 'rhel-8-1']:
```

Comment:
Has this already reached our images? (4 days ago it was not in our images as I pointed [here](https://github.com/cockpit-project/cockpit/pull/11495#issuecomment-478093083)

Reply:
Ah, don't mind me :) I saw you put it in ` test/verify/machineslib.py`

Reply:
It won't ever. This doesn't assume a change in our images, it just moves the expected message into machineslib.py below, so that it only applies to one test (`testCreate`).